config/webhook/kustomizeconfig.yaml

```diff
@@ -0,0 +1,25 @@
+# the following config is for teaching kustomize where to look at when substituting vars.
```

Comment:
There are no webhooks that are part of CLO, we should remoe

Reply:
done